BridgeAppSDK/SBAOnboardingStepController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SBAOnboardingStepController.swift
+//  BridgeAppSDK
+//
+//  Copyright © 2016-2017 Sage Bionetworks. All rights reserved.
```

Comment:
It was renamed (though I forget the original name). It's only used during onboarding and it had a name that was confusingly similar to the name of a protocol or class in ResearchUXFactory.